src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+<Project>
+
+  <Target
+    Name="_BlazorCopyFilesToOutputDirectory"
+    DependsOnTargets="BlazorBuild"
+    Inputs="@(BlazorItemSpec)"
+    Outputs="@(BlazorItemSpec->'%(TargetOutputPath)')"
+    AfterTargets="CopyFilesToOutputDirectory">
+
+    <!-- Copy the blazor output files  -->
+    <Copy
+      SourceFiles="@(BlazorItemSpec)"
+      DestinationFiles="@(BlazorItemSpec->'%(TargetOutputPath)')"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)"
+      UseSymboliclinksIfPossible="$(CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible)"
+      Condition="'@(BlazorItemSpec)' != '' and '$(CopyBuildOutputToOutputDirectory)' == 'true' and '$(SkipCopyBuildProduct)' != 'true'">    
+    </Copy>
+
+    <ItemGroup>
+      <BlazorOutputs Include="$(ProjectDir)$(OutputPath)dist/**/*" />
```

Comment:
Can we leverage `CopiedFiles`? https://msdn.microsoft.com/en-us/library/3e54c37h.aspx

Generally listing files using `**/*` is for users not for tool authors. There are gotchas here.

Reply:
This is just for verbose reporting only. I've changed it to use a transform. CopiedFiles wouldn't work due to target batching